src/libraries/System.Private.CoreLib/src/System/Activator.RuntimeType.cs

```diff
@@ -12,6 +12,10 @@ namespace System
 {
     public static partial class Activator
     {
+        //
+        // Note: CreateInstance returns null for Nullable<T>, e.g. CreateInstance(typeof(int?)) returns null.
```

Comment:
We should add that as a remarks in the API docs. Look like useful info.

https://docs.microsoft.com/en-us/dotnet/api/system.activator.createinstance?view=netcore-3.1#System_Activator_CreateInstance_System_Type_System_Reflection_BindingFlags_System_Reflection_Binder_System_Object___System_Globalization_CultureInfo_System_Object___

cc @mairaw, @carlossanlop

Reply:
Done: https://github.com/dotnet/dotnet-api-docs/pull/3718